core/capabilities/webapi/trigger.go

```diff
@@ -239,9 +241,20 @@ func (h *triggerConnectorHandler) UnregisterTrigger(ctx context.Context, req cap
 	return nil
 }
 
+func (h *triggerConnectorHandler) Info(ctx context.Context) (capabilities.CapabilityInfo, error) {
```

Comment:
Is there some interface that we can implement that assures the functions we require are present?  It seems strange that this was easy to miss.